docs/admin/system-information.rst

```diff
@@ -1956,57 +1956,67 @@ Snapshots
+| ``started``              | The point in time when the       | ``TIMESTAMP WITH TIME ZONE`` |
+|                          | creation of the snapshot         |                              |
+|                          | started. Changes made after      |                              |
+|                          | that are not stored in this      |                              |
+|                          | snapshot.                        |                              |
++--------------------------+----------------------------------+------------------------------+
+| ``finished``             | The point in time when the       | ``TIMESTAMP WITH TIME ZONE`` |
+|                          | snapshot creation finished.      |                              |
++--------------------------+----------------------------------+------------------------------+
+| ``state``                | The current state of the         | ``TEXT``                     |
+|                          | snapshot. One of:                |                              |
+|                          | ``IN_PROGRESS``, ``SUCCESS``,    |                              |
+|                          | ``PARTIAL``, or ``FAILED``.      |                              |
++--------------------------+----------------------------------+------------------------------+
+| ``version``              | An internal version this         | ``TEXT``                     |
+|                          | snapshot was created with.       |                              |
++--------------------------+----------------------------------+------------------------------+
+| ``failures``             | A list of failures that occurred | ``ARRAY(TEXT)``              |
+|                          | while taking the snapshot.       |                              |
+|                          | If taking the snapshot was       |                              |
+|                          | successful this is empty.        |                              |
++--------------------------+----------------------------------+------------------------------+
+| ``reason``               | Snapshot failure reason.         | ``TEXT``                     |
+|                          | If taking the snapshot was       |                              |
+|                          | successful this is ``NULL``.     |                              |
++--------------------------+----------------------------------+------------------------------+
+| ``total_shards``         | Total number of shards in the    | ``INTEGER``                  |
+|                          | snapshot.                        |                              |
++--------------------------+----------------------------------+------------------------------+
+| ``include_global_state`` | Flag indicating that snapshot    | ``BOOLEAN``                  |
+|                          | was taken with ``ALL`` option.   |                              |
```

Comment:
Max width changed because of this column, hence adjustments in the first column length for the whole table.